Shared/Flows/Waitlist/WaitlistViewController.swift

```diff
@@ -0,0 +1,140 @@
+//
+//  WaitlistViewController.swift
+//  Jibber
+//
+//  Created by Martin Young on 1/10/22.
+//  Copyright © 2022 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import StreamChat
+import Parse
+
+class WaitlistViewController: DiffableCollectionViewController<MessageSequenceSection,
+                             MessageSequenceItem,
+                             MessageSequenceCollectionViewDataSource> {
```

Comment:
This VC will show the waitlist conversation that the person was assigned to. Because we don't have that set up yet, there's just a blank space where the conversation will be.